metricshub-snmpv3-extension/src/test/java/org/sentrysoftware/metricshub/extension/snmpv3/SnmpV3ExtensionTest.java

```diff
@@ -517,6 +517,18 @@ void testIsValidConfiguration() {
 				new IConfiguration() {
 					@Override
 					public void validateConfiguration(String resourceKey) throws InvalidConfigurationException {}
+
+					@Override
+					public String getHostname() {
+						// TODO Auto-generated method stub
```

Comment:
Remove comment or add a real to do if the intent is to update something in the future.